src/request/sign-transaction/SignTransaction.css

```diff
@@ -1,154 +1,300 @@
+#confirm-transaction {
+    position: relative;
+}
+
 #layout-container {
+    overflow: hidden;
+}
+
+#layout-container.standard .hide-standard,
+#layout-container.checkout .hide-checkout {
+    display: none;
+}
+
+#layout-container .nq-card-footer {
+    border: 0;
+    padding: 3rem 4rem;
+}
+
+#layout-container .nq-card-body {
+    padding: 2rem 4rem 0rem;
```

Comment:
If overwriting the padding, remember to also declare mobile styles with reduced paddings for screens smaller than 450px, like the original styles do. Or better, only overwrite the paddings that you need, such as `padding-top` and `padding-bottom`.